Migration.md

```diff
@@ -5,6 +5,13 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Gazebo Physics 8.X to 9.X
+
+### Removals
+
+* The environment variable `GZ_PHYSICS_ENGINE_INSTALL_DIR` is removed. Use
```

Comment:
rephrased to say macro instead of env var

```suggestion
* The macro `GZ_PHYSICS_ENGINE_INSTALL_DIR` is removed. Use
```

Reply:
Updated in [97f5255](https://github.com/gazebosim/gz-physics/pull/705/commits/97f52559839befde01e5ac4ded1aae1477f24fbf).